### PR TITLE
Feature/cleanup

### PR DESCRIPTION
**Pull request by ladydascalie**
This provides a bunch of improvements to how the documentation displays in GoDoc (and gets rid of redundant information)

This also renames `formatters.go` and `routes.go` to names matching their actual package name

### CODE REVIEW

File: format/format.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// Pckage format defines any necessary custom formatters for the responses
```
Comment (danmrichards) [inline format/format.go]: Typo (Pckage -> Package)

File: pagination/paginator.go [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// Package pagnination defines a paginator able to return formatted responses
```
Comment (danmrichards) [inline pagination/paginator.go]: Typo (pagnination -> pagination)

**Review by danmrichards — CHANGES_REQUESTED** [review]

**Comment by danmrichards** [comment]
Just a couple typos @LadyDascalie then LGTM

**Comment by ladydascalie** [comment]
typos fixed

**Review by danmrichards — APPROVED** [review]

**Pull request merged**